src/main/scala/stuff/monads.scala

```diff
@@ -0,0 +1,101 @@
+package ohnosequences.stuff
+
+import naturalTransformations._
+import products._
+import functions._
+
+abstract
+class Monad {
+
+  type On <: Functor.endo
+  val on   : Functor.is[On]
+
+  // TODO do I need another Functor.is here?
+  val μ: Functor.Composition[Functor.is[On],Functor.is[On]] ~> Functor.is[On]
+
+  val ι: Functor.is[Functor.Identity[On#Source]] ~> Functor.is[On]
+}
```

Comment:
I'm not familiar with category theory notation, but the places I read about monads noted these natural transformations as η and μ. Is this syntax also usual?

Reply:
Yes, that's also a common convention. I tend to prefer `ι` as this *is* a unit for a monoid, after all. If you think `η` would be better (`ι ≃ i - ˙` tends to generate some confusion), it can be changed; it matches the adjunction conventions too.

Reply:
Either way is good for me, I just wanted to know if this was a custom notation :)